Week-1/Week-1

```diff
@@ -1 +1 @@
-
+Hi I am Harsh Vardhan.
```

Comment:
The readme file for each week can be about the exercises worked on.

Reply:
Yes done.